docs/src/main/asciidoc/datasource.adoc

```diff
@@ -253,7 +253,12 @@ Without an extension, the driver will work correctly in any Quarkus app running
 However, the driver is unlikely to work when compiling your application to a native executable.
 If you plan to make a native executable, use the existing JDBC Quarkus extensions, or contribute one for your driver.
 
-.An example with the OpenTracing driver:
+[NOTE]
```

Comment:
This should probably be a warning, right?

Reply:
agree